authors/apps/article/tests/test_rate_article.py

```diff
@@ -53,21 +61,25 @@ def setUp(self):
         body = "this is a body"
         author = self.user
         article = Article(
-            author=author,
+            user=author,
             slug=self.slug,
             body=body,
             title=title,
-            description=description)
+            description=description
+        )
         article.save()
         self.rate_details = {
             "user": {
                 "slug": self.slug,
                 "rate": 3
             }
         }
-        self.rate_url = os.environ["URL"]+"api/article/"\
-            + self.slug + "/rate/"
-        self.view_rates_url = os.environ["URL"]+"api/article/rate/"
+        self.rate_url = os.environ["URL"] + \
```

Comment:
Why use the environment variables to pass the URLs instead of building your URLs using Django inbuilt reverse mechanism. I don't think this is a good thing.

Reply:
This is Gidrafs work that has already been merged.  I just made some soft touches to it to make sure it worked together with my work. You should have raised this at Gidrafs pr.

Reply:
👍